charter/index.html

```diff
@@ -167,40 +172,41 @@ <h1 id="title">PROPOSED Solid Working Group Charter</h1>
 
       <div id="background" class="background">
         <h2>Motivation and Background</h2>
-        <p>It is common for web service providers to require users to surrender control over their own data, and store them in a way that is often restricted to certain systems, or can only be accessed by the applications that created it.</p>
-
-        <p>Solid is largely based on existing open web standards, and extends them to give priority to individual and community autonomy. It allows them to keep authority over their identity, data, and privacy, and gives them the freedom to select applications and services that suit their needs.</p>
-
-        <p>Solid defines the concept of storage (one feature of the construct informally referred to as Solid Pods), in which users place data and control access to it, and a suite of protocols for applications to use storages.</p>
-
-        <p>Solid offers several advantages over more traditional architectures for data use by web services today, including:</p>
+        <p>It is common for web service providers to require users to surrender control over their own data, and store them in a way that is often restricted to certain systems, or can only be accessed by the applications that created them.
+        </p>
 
-        <ul>
-          <li>Storage is controlled by individuals or organizations, and can be delegated to other parties.</li>
-          <li>Interoperability between applications and various storage server implementations of the protocol.</li>
-          <li>A decentralized trust model enabling authentication and authorization mechanisms across the web.</li>
-        </ul>
+        <p><span class=new>A number of initiatives — such as <a href="https://indieweb.org/">IndieWeb</a>, <a href="https://www.w3.org/community/unhosted/">Unhosted</a>, and <a href="https://solidproject.org/">Solid</a> — have emerged to propose alternative models for web applications.
+        They are</span> largely based on existing open web standards, which they extend to give priority to individual and group autonomy.</p>
+        
+        <p>The <a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a> has been incubating <a href="https://solidproject.org/TR/">specifications</a> since 2018 towards that end, i.e., <q cite="https://www.w3.org/groups/cg/solid">to describe the interoperability between different classes of products by using web communication protocols, global identifiers, authentication and authorization mechanisms, data formats and shapes, notifications, and query interfaces.</q>
```

Comment:
"global identifiers" doesn't seem to be something that is worth being proud of about?

Reply:
Hi @samuelgoto, I would find it interesting to compare the use of WebID as a global identifier to the use of email addresses as a global identifier.  In OIDC, `email` scope is very commonly used, and  [FedCM has email as part of user info](https://fedidcg.github.io/FedCM/#dom-identityuserinfo-email).
Probably, we should pick a better venue for that comparison than this inline comment.

Reply:
Email addresses can be directed/sharded, rather than global.

Reply:
People were looking into potentially similar approaches with WebId, such as https://kushaldas.in/categories/webid.html.
It would be great to document those practices in one place eventually. From my current standpoint, email addresses and WebIDs should be pretty comparable as identifiers. 
Ultimately, people should be able to do what fits their needs, while correlatable shouldn't be the only option. Many individuals, probably even more organizations, want to establish a strong web identity. Even within the dev community, many people try to register the same handle across many online services.
Thinkin' about what can be realistically addressed in this or one of the next PRs here, maybe we should at least mention that the proposed WG should address known issues with correlatable identifiers. /cc @michielbdejong

Reply:
I see people using IDs for Solid on a wide scale, between totally public like your linked in profile (I [wrote about recently](https://www.w3.org/DesignIssues/PersonalPublic.html) )  to single-use large random number IDs for whistle-blowing. In between, medium security IDs like alumni, club, and work emails, where some people know the correlation and some don't.  Having a variety of IDPs o different types helps that. 

It is important that this point in the Solid architecture is an extension pont so we can play with different authentication protocols, and support more than one at a time.

Reply:
@samuelgoto — Note that there is no restriction that any agent (especially, human) may only have _n_ (which I think you read as _1_) ["global identifiers"](https://github.com/solid/solid-wg-charter/pull/69/files#r1529372222) (e.g., WebIDs) for themselves. I could have one WebID that I use for all my Rebel Alliance interactions, another that I use for all my Galactic Republic interactions, another that I use for all my Jedi interactions, and yet another that I use for all my Galactic Empire interactions. If and when I choose to announce that any two of these are co-referent to "me", I may choose to do so in one of those arenas, but remain distinct in the others. Key points are that the assertion/implication of coreference is my decision, and that none of these incorporate a mandated "phone home" except in the course of dereferencing that WebID in order to confirm the user's assertion of control therof — which is a known activity.

"Global identity" *can* be "something that is worth being proud of", especially as it allows pseudonymous activity that is interpreted only based on that activity, and not based on the fact that, e.g., the pseudonymous author is 12 years old. (See some discussion of this in OSC's Ender series, particularly e.g., _[Valentine Wiggin](https://en.wikipedia.org/wiki/Valentine_Wiggin), under her pseudonym [Demosthenes](https://en.wikipedia.org/wiki/Demosthenes), publishes_.)

Reply:
@samuelgoto — Also note that WebIDs can also be "directed/sharded" (i.e., they can indirectly identify an agent-in-a-role, like "CEO of Company_X" or a group-that-acts-as-a-singular, like "BoD of Company_X" where any Board member can act on the Board's behalf).

Reply:
Maybe you should call these "unique" rather than "global" then?

Reply:
They're "global" because (HTTP/S-)URI-based identifiers (like WebID) offer dereferenceability as a feature, always being resolvable to descriptions of the identified agent/entity. This dereferenceability is available anywhere, anytime, to anyone — though there might be some dereferences which require authentication by the caller in order to get that description, and some which can only be completed when the caller is within a VPN or similar, etc.

As to "unique", that refers to the identified agent/entity (i.e., the identifier uniquely identifies an agent/entity. People often get confused, and think the WebID identifier cannot be lexically repeated, as with UUIDs/GUIDs), and is only guaranteed within a timestamp, because domain registrations change, among other things. Within that timestamp, however, they are intended to uniquely identify an entity/agent, such that anyone who refers to a given WebID (or similar) is understood to intend to mean the same entity/agent. 

Outside the bounds of the indeterminate timestamp, the agent/entity identified by the WebID (or similar) may differ — but the descriptions of the "earlier" and "later" entities/agents are expected to include sufficient detail — such as timestamps of description creation, modification, termination — that the dereferencer can handle each